processing/src/main/java/io/druid/query/aggregation/cardinality/CardinalityAggregatorFactory.java

```diff
@@ -40,7 +40,7 @@
 import java.util.Comparator;
 import java.util.List;
 
-public class CardinalityAggregatorFactory implements AggregatorFactory
+public class CardinalityAggregatorFactory extends AggregatorFactory
```

Comment:
CardinalityAggregatorFactory's combining factory is HyperUniquesAggregatorFactory, so same will never be stored inside metadata . However, changed it to throw unsupported op exception if merging is ever called.